src/products/firefox_android/how_to_localize.md

```diff
@@ -1,35 +1,17 @@
 # Firefox for Android - How to localize
 
-The release cycle for Firefox for Android is the same as Firefox for Desktop, which is generally 6 weeks - but can sometimes be 8 weeks.
+Firefox for Android is on a four week rapid release cycle. This means that usually there is a new release going out every four weeks.
 
-You can see the [detailed schedule for each release here](https://wiki.mozilla.org/RapidRelease/Calendar).
+Within this timeframe, localizers will usually have four weeks to localize and test their strings for a given release: strings land continuously during the first two weeks, and then stop landing for two weeks so that you can complete your work. The deadline in Pontoon is an indicator of when the next localization cut-off date is for the upcoming release.
 
-*Merge day* is when localizations move to the next train - and the last day available to include localization updates to the next Beta version if you work on tools like Pontoon.
+## Localize Firefox for Android in Pontoon
 
-Note: please make sure that you’ve already localized mozilla.org before you start working on Firefox for Android or any other projects.
+* If your locale has never been set up for any android-l10n project yet, you must fill out [this issue template](https://github.com/mozilla-l10n/android-l10n/issues/new?assignees=delphine&labels=&template=new-localization-request.md&title=Add+%5Blocale%5D+to+%5Bproduct%5D) in order to request to add your locale to “Fenix” (code name for Firefox for Android). You will be notified there once your locale is set up on Pontoon and is ready to be localized.
 
-## Localize the Firefox for Android folder
+Note: android-l10n is a project in which live several mobile products, that all share common strings contained in files called “Android Components”. Android Components (also internally called a-c) are a collection of Android libraries to build browsers or browser-like applications. The strings in Android Components serve various mobile products such as Fenix, Firefox Reality, Lockwise and Fire TV. To simplify things, we’ve created [tags here in Pontoon](https://pontoon.mozilla.org/projects/android-l10n/tags/), so you can easily access all the product-specific strings needed, as well as the a-c ones (that are required as well for the products).
 
-* [Pontoon](https://pontoon.mozilla.org/projects/firefox-for-android/): choose your locale code from there.
+* Once your locale has been set up, choose it from the list of locales on [Pontoon](https://pontoon.mozilla.org/projects/android-l10n/tags/fenix/).
```

Comment:
"on Nightly" or "in Nightly"?

Reply:
Made that correction in a few instances throughout docs